bin/spring

```diff
@@ -3,15 +3,15 @@
 # This file loads spring without using Bundler, in order to be fast.
 # It gets overwritten when you run the `spring binstub` command.
 
-unless defined?(Spring)
-  require 'rubygems'
-  require 'bundler'
+# unless defined?(Spring)
```

Comment:
@kirkkwang Did we mean to permanently comment this out?

Reply:
oh no we shouldn't